backend/private-graph/graph/schema.resolvers.go

```diff
@@ -402,11 +401,15 @@ func (r *mutationResolver) UpdateAdminAboutYouDetails(ctx context.Context, admin
 	admin.UserDefinedRole = &adminDetails.UserDefinedRole
 	admin.Referral = &adminDetails.Referral
 	admin.UserDefinedPersona = &adminDetails.UserDefinedPersona
-	admin.Phone = adminDetails.Phone
+	admin.Phone = pointy.String("")
 	admin.AboutYouDetailsFilled = &model.T
 
 	if !util.IsDevEnv() {
 		r.PrivateWorkerPool.SubmitRecover(func() {
+			// Delay this because we want the DB transaction creating the workspace
+			// (UpdateAdminAndCreateWorkspace) to complete before this executes.
+			time.Sleep(10 * time.Second)
```

Comment:
This is kind of janky. I'd like to figure out a better way of triggering these. I explored using GORM callbacks, but that would have been a larger refactor because we don't have access to the resolver there. Open to ideas how to improve this 😅